.travis.fedora.sh

```diff
@@ -11,15 +11,26 @@ make package
 # /proc and /sys in the chroot).
 mkdir -p /root/rpmbuild/SOURCES
```

Comment:
*(Out of scope of this PR: I just noticed that maybe we should use `rpm --eval "%_topdir"` or something like that instead of hardcoding the `/root/rpmbuild/...` path...)*

Reply:
`%_topdir` is what I was looking for, thanks!